common/src/main/java/com/revenuecat/purchases/common/subscriberattributes/SpecialSubscriberAttributes.kt

```diff
@@ -1,40 +1,40 @@
-package com.revenuecat.purchases.subscriberattributes
+package com.revenuecat.purchases.common.subscriberattributes
```

Comment:
the reasoning behind changing the module is that this now needs to be accessed from the `google` and `amazon` module